.github/workflows/release.yml

```
@@ -0,0 +1,52 @@
+        run: npm run build
+
+      - name: Test
+        run: npm run test
+
+      - name: Configure git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Bump version
+        run: npm version "${{ inputs.version }}" -m "chore(release): %s"
```

actionlint (1.7.10)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)